### PR TITLE
Cleanup Client Directory

### PR DESCRIPTION
**Pull request by holeyness**
Moved JS file into js folder, updated tsconfig to support that.

### CODE REVIEW

**Comment by ethanpailes** [comment]
LGTM

**Review by ethanpailes — APPROVED** [review]

File: app/src/ts/app.ts [hunk 1]
```diff
@@ -15,8 +15,11 @@ function createWindow(): void {
     win = new BrowserWindow({ width: 800, height: 600 });
 
     console.log(`${__dirname}`);
-    win.loadURL(`file://${__dirname}/index.html`);
 
+    // Load Index, I did this to accomadate reorganizing the js dir
+    win.loadURL('file://' + __dirname + '/../index.html');
```
Comment (GrahamGoudeau) [inline app/src/ts/app.ts]: Use ES6 string interpolation with ` ${} `

File: app/src/ts/upload.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@
 // The script for uploading a file
 
 import * as $ from "jquery";
+const remote = require('electron').remote;
```
Comment (GrahamGoudeau) [inline app/src/ts/upload.ts]: Is this the best way to import this? Is there no way to use `import { } from 'electron'` syntax?

**Review by GrahamGoudeau — COMMENTED** [review]
Some good software engineering lessons in here

**Comment by GrahamGoudeau** [comment]
LGTM

**Review by ethanpailes — APPROVED** [review]

**Review by GrahamGoudeau — APPROVED** [review]

**Pull request merged**